client/hosting/hosting-features/components/hosting-features.tsx

```diff
@@ -61,32 +59,14 @@ const HostingFeatures = () => {
 
 	const hasEnTranslation = useHasEnTranslation();
 
-	const { data: siteTransferData, refetch: refetchSiteTransferData } = useSiteTransferStatusQuery(
-		siteId || undefined
-	);
-	// `siteTransferData?.isTransferring` is not a fully reliable indicator by itself, which is why
-	// we also look at `siteTransferData.status`
-	const isTransferInProgress =
+	const { data: siteTransferData } = useSiteTransferStatusQuery( siteId || undefined, {
```

Comment:
```suggestion
	const { data: siteTransferData, isLoading: isLoadingTransferData } = useSiteTransferStatusQuery( siteId || undefined, {
```
I realize it would make sense to display a loading placeholder on this page while fetching this data. 

In the interest of managing scope, I guess we could do that in a follow-up PR, though.